src/models.py

```diff
@@ -202,6 +203,7 @@ def __str__(self):
 
 permissions_api_keys = Table(
     'permissions_api_keys',
+    BaseModel.metadata,
```

Comment:
this shouldnt be here, also will cause merge error

Reply:
changed per above.